scripts/c2r_script.py

```diff
@@ -952,9 +953,9 @@ def main():
             logger.info("Cleaning up modifications to the system.")
             cleanup(required_files)
 
-        print("### JSON START ###")
-        print(json.dumps(output.to_dict(), indent=4))
-        print("### JSON END ###")
+        logger.info("### JSON START ###")
+        logger.info(json.dumps(output.to_dict(), indent=4))
+        logger.info("### JSON END ###")
```

Comment:
Since tasks backend specifically searches for these to capture the output, what happens whe nI set log level to WANR? then these separators won't be i nthe output and I think task backend wouldn't be able to handle it. Since this output is something the script should do always, shouldn't we keep the print?

Reply:
Hmmm, good point. I just replaced for every print() and replaced with logger.info. 

I will switch it back to print

Reply:
```suggestion
        print("### JSON START ###")
        print(json.dumps(output.to_dict(), indent=4))
        print("### JSON END ###")
```